com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -405,15 +435,13 @@ class Job(_ResourceElement):
         healthy
     """
     def get_application_logs(self, path=None, prefix=None):
-        """Retrieves the application log and trace files of the job and saves them as a tar file to the specified path
-        with the given name.
+        """Retrieves the console logs and application traces of the job's processing elements (PEs) and stores them in a compressed 'tar' archive (that is, a '.tar.gz' file) with the specified file name in the specified directory path.
```

Comment:
Nit: try to keep documentations lines less than 80 characters. It affects how documentation is displayed in editors like PyCharm.

Reply:
I'm not familiar with PyCharm. I've reformatted the method documentation to 80-character lines. If that messes up the paragraph flow in generated HTML, let me know and I'll try something different.